ghost/collections/src/CollectionsService.ts

```diff
@@ -401,7 +401,11 @@ export class CollectionsService {
             filter: `posts:${postId}`
         });
 
-        return collections.map(collection => this.toDTO(collection));
+        return collections.map(collection => this.toDTO(collection))
+            .sort((a, b) => {
```

Comment:
Should this be happening at the repository level? 🤔

Reply:
true. ordering is respositorie's concert 👍

Reply:
Note, when moved to the repository level there was a race condition happening in the post's event processing. So leaving this as is for current purposes, but will need to move this logic once the event processing is improved.